CloudSQLReplicator/tests/util/cloudsql_test.py

```diff
@@ -0,0 +1,323 @@
+# Copyright 2018 The Pontem Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests GCP API Cloud SQL utility functions."""
+
+import unittest
+import mock
+from mock import MagicMock
+import uuid
+
+import google.auth
+from google.oauth2 import credentials
```

Comment:
The style guide mentions: Use import statements for packages and modules only, not for individual classes or functions. Ditto for other 'from' imports.

Reply:
Fixed.
credentials is from a separate [module](https://github.com/googleapis/google-auth-library-python/blob/master/google/oauth2/credentials.py).